lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -322,11 +323,19 @@ def __init__(self, *args, **kwargs):
 
     def _gen_axes_patch(self):
         # docstring inherited
-        return mpatches.Polygon(self.get_grid_helper().get_boundary())
+        # Using a public API to access _extremes.
+        (x0, _), (x1, _), (y0, _), (y1, _) = map(
+            self.get_grid_helper().get_data_boundary,
+            ["left", "right", "bottom", "top"])
```

Comment:
Do we want to introduce `get_data_boundary()` without parameters as returning all four; in analogy to `get_boundary()`?

Reply:
I think I would rather fix FloatingAxes so to make `get_xlim()/get_ylim()` and even `set_xlim()/set_ylim()` work, rather than introducing additional ad-hoc APIs.